src/ride/ride_ratings.c

```diff
@@ -599,17 +599,214 @@ static int sub_65E72D(rct_ride *ride)
 	return edx & 0xFFFF;
 }
 
+static rating_tuple get_var_10E_rating(uint16 var_10E) {
+	int al;
```

Comment:
var_10E would appear to be of the following format
`XXXX_Xaaa_bbbc_cccc`

Where:
a = unk_1
b = unk_2
c = unk_3

Make a function for accessing each of these variables and then use that in this function. 

```
int get_var_10E_unk_1(rct_ride* ride){
    return (ride->var_10E >> 8) & 0x7;
}

int get_var_10E_unk_2(rct_ride* ride){
    return (ride->var_10E >> 5) & 0x7;
}

int get_var_10E_unk_3(rct_ride* ride){
    return ride->var_10E & 0x1F;
}
```